src/utils/temporal.cpp

```diff
@@ -175,6 +176,10 @@ int64_t Date::MicrosecondsSinceEpoch() const {
 
 int64_t Date::DaysSinceEpoch() const { return utils::DaysSinceEpoch(year, month, day).count(); }
 
+std::string Date::ToString() const {
+  return fmt::format("{:0>4}-{:0>2}-{:0>2}", year, static_cast<int>(month), static_cast<int>(day));
```

Comment:
This was copied from the previous implementation of `operator<<`. But I am guessing because `month` and `day` are declared as `uint8_t` which is formatted as a character by `fmt::format()`

Reply:
Yes, can confirm.